vnext/Microsoft.ReactNative/Views/ViewViewManager.cpp

```diff
@@ -385,19 +395,35 @@ bool ViewViewManager::UpdateProperty(
     }
   }
 
-  if (auto view = pViewShadowNode->GetView().try_as<xaml::UIElement>()) {
+  return ret;
+}
+
+void ViewViewManager::OnPropertiesUpdated(ShadowNodeBase *node) {
+  auto *viewShadowNode = static_cast<ViewShadowNode *>(node);
+  auto panel = viewShadowNode->GetViewPanel();
+
+  if (panel.Background() == nullptr) {
+    // In XAML, a null background means no hit-test will happen.
+    // We actually want hit-testing to happen if the app has registered
+    // for mouse events, so detect that case and add a transparent brush.
+    if (viewShadowNode->IsHitTestBrushRequired()) {
+      panel.Background(EnsureTransparentBrush());
```

Comment:
curious why we need to keep a reference to the brush in the `ViewViewManager`, i.e. why not just create the brush here and pass it to the `Background` property?

Reply:
This was for perf reasons